composer.json

```diff
@@ -22,7 +22,7 @@
         }
     },
     "require": {
-        "php": ">=5.6.5"
+        "php": ">=5.6.5 || >=7.0"
```

Comment:
apparently it was related to composer itself, after an update this issues does not appear anymore